macros/cross_db_utils/datediff.sql

```diff
@@ -16,8 +16,8 @@
 
 {% macro bigquery__datediff(first_date, second_date, datepart) %}
 
-    date_diff(
-        {{second_date}},
+    datetime_diff(cast(
```

Comment:
same here, let's use timestamps instead of datetimes. I imagine we'll want to cast both the first and second date below, right?

Reply:
same thing here...maybe we should talk a little more when you have time to discuss these...i might be getting the wrong idea about how the tests should work, but my general thought process around these date/time casts is based on how the result should look in the end